docker/vendor/nginx-es/Dockerfile

```diff
@@ -13,7 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM astronomerinc/ap-base:0.7.5
+# TODO: replace to 0.10.1
```

Comment:
@andriisoldatenko maybe we should make this base image just the latest stable Alpine instead of ap-base.

Reply:
fixed.